constants/lilyWeight.js

```diff
@@ -0,0 +1,966 @@
+module.exports = {
```

Comment:
If you add lily weight please use the lilyweight npm package (https://www.npmjs.com/package/lilyweight) its easier to update